ui/apps/portal/src/plugins/SystemAdmin/StudyPage/StudyCard/StudyCard.scss

```diff
@@ -0,0 +1,114 @@
+.study-card {
+  width: 824px;
+  height: 176px;
+  margin: 2px 0;
+  cursor: pointer;
```

Comment:
NIT: would prefer if study card does not have `cusor: pointer` because it implies to the user that the whole card can be clicked.